src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs

```diff
@@ -556,7 +557,8 @@ static ImmutableArray<FunctionPointerParameterSymbol> makeParametersFromMetadata
                         var paramRefCustomMods = CSharpCustomModifier.Convert(param.RefCustomModifiers);
                         var paramType = TypeWithAnnotations.Create(param.Type, customModifiers: CSharpCustomModifier.Convert(param.CustomModifiers));
                         RefKind paramRefKind = getRefKind(param, paramRefCustomMods, RefKind.In, RefKind.Out);
-                        paramsBuilder.Add(new FunctionPointerParameterSymbol(paramType, paramRefKind, i, parent, paramRefCustomMods));
+                        DeclarationScope scope = DeclarationScope.Unscoped; // PROTOTYPE: Include scope.
```

Comment:
>  DeclarationScope scope = DeclarationScope.Unscoped; // PROTOTYPE: Include scope. [](http://example.com/codeflow?start=22&length=82)

I think it would be better to not make changes to this symbol for now. All of them effectively make ```FunctionPointerParameterSymbol``` to always return ```DeclarationScope.Unscoped```. We can return this value unconditionally from ```FunctionPointerParameterSymbol```, and if that is not the default ```DeclarationScope``` for all scenarios, add a PROTOTYPE comment to adjust implementation there.